.github/workflows/ci.yml

```diff
@@ -99,8 +99,23 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
-        nautobot-version: ["1.2.1"]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        db-backend: ["postgresql"]
```

Comment:
Should add this as well. Would be good to see if that syntax works, (open question in cookie cutter anyway) 
```
      - name: "Use Mysql invoke settings when needed"
        run: "cp invoke.mysql.yml invoke.yml"
        if: "matrix.db-backend == 'mysql'"
```